spec/features/users_index_spec.rb file

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'User', type: :feature do
+  before :each do
```

Comment:
- [ ] Also, After running the tests I noticed there are some tests that are not passing, Please make sure all tests are passing before the next review request. 

### The issue 
<img width="1680" alt="Screenshot 2023-11-16 at 11 32 28 in the morning" src="https://github.com/mohammadnewton/blog_app/assets/69077061/9e6e1925-40fc-482f-8fac-b9998c80cd3a">